packages/react/src/components/Notification/Notification.tsx

```diff
@@ -976,7 +998,17 @@ export function ActionableNotification({
       ref={ref}
       role={role}
       className={containerClassName}
-      aria-labelledby={title ? id : subtitleId}>
+      aria-labelledby={title ? id : subtitleId}
+      onBlur={handleBlur}
+      tabIndex={hasFocus ? 0 : -1}>
```

Comment:
This is causing an accessibility error

> The tabbable element's role 'alertdialog' is not a widget role

I don't think this is needed? I think when the component is rendered, focus should be placed on the action/button, not on the container.

Reply:
Now it is focusing on the action button